mongomock/collection.py

```diff
@@ -2050,7 +2052,7 @@ def _current_date_updater(doc, field_name, value):
         if value == {'$type': 'timestamp'}:
             doc[field_name] = helpers.get_current_timestamp()
```

Comment:
Note that we should also update this one somehow. Please add a TODO or find a way already.

Reply:
I'll try and revisit this separately